src/scope.ts

```diff
@@ -234,7 +234,7 @@ export function getScopeFunction(key: string, withNew: boolean): Function {
 }
 
 export function isMethodWhitelisted(member: string, property: string): boolean {
-  if (ALLOWED_CLASS_EXPRESSIONS[member]) {
+  if (ALLOWED_CLASS_EXPRESSIONS.hasOwnProperty(member)) {
```

Comment:
We probably just don't have the same eslint config here as elsewhere so this is not highlighted, but [you generally don't want to call prototype builtins on the objects](https://eslint.org/docs/latest/rules/no-prototype-builtins) like that

```suggestion
  if (Object.prototype.hasOwnProperty.call(ALLOWED_CLASS_EXPRESSIONS, member)) {
```